src/module.cc

```diff
@@ -1,5 +1,6 @@
-#include "node_report.h"
+// Copyright 2017 Nodereport
```

Comment:
@jasnell  you may be bringing all Node.js src copyright statements into alignment with legal requirements, opinions on this?